GoogleUtilities.podspec

```diff
@@ -98,4 +98,11 @@ other Google CocoaPods. They're not intended for direct public usage.
     ud.private_header_files = 'GoogleUtilities/UserDefaults/Private/*.h'
     ud.dependency 'GoogleUtilities/Logger'
   end
+
+  s.test_spec 'unit' do |unit_tests|
+    # All tests require arc except Tests/Network/third_party/GTMHTTPServer.m
+    unit_tests.source_files = 'GoogleUtilities/Example/Tests/**/*.[mh]'
+    unit_tests.requires_arc = 'GoogleUtilities/Example/Tests/*/*.[mh]'
+    unit_tests.dependency 'OCMock'
```

Comment:
I wonder, if something like 
`unit_tests.xcconfig = {'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) GUL_APP_DELEGATE_TESTING=1 }`
can replace the definition
```
#ifdef DEBUG
#define GUL_APP_DELEGATE_TESTING
#endif
```

Reply:
I'd rather not mess with options outside of the source if not necessary.  I don't see an issue with including test only code in DEBUG builds. In fact, I'm not even sure we should keep GUL_APP_DELEGATE_TESTING and GUL_UNSWIZZLING_ENABLED.

Reply:
SGTM. Yeah, I would prefer to substitute GUL_APP_DELEGATE_TESTING by just DEBUG.